### DISCUSSION TITLE
台灣之光（？

### DISCUSSION THREAD

**Discussion by ayakasanma** [1101-GS3073-C/Python-Community-GS3073-C#205] [科技新知]
目前資料看起來天璣9000是可以和高通Snapdragon 8 Gen1打對台的！難道以後安卓的高階旗艦機有機會能看到聯發科的CPU了嗎？
https://ctee.com.tw/news/tech/580101.html

**Comment by Shiba010** [comment]
聯發科要成為台灣第二座護國神山了嗎？！
